src/bundle/Resources/public/js/scripts/admin.main.translation.update.js

```diff
@@ -0,0 +1,10 @@
+(function(global, doc) {
+    const updateMainTranslation = (event) => {
+        doc.querySelector('#main_translation_update_language_code').value = event.target.value;
+        doc.querySelector('form[name="main_translation_update"]').submit();
+    };
+
+    [...doc.querySelectorAll('input[name="updateMainTranslation"]')].forEach((input) =>
```

Comment:
There is no longer need to use `[...` with `querySelectorAll`. We used to add this because in older versions of Edge forEach was not supported on list returned by `querySelectorAll`.

Reply:
I see there are multiple naming conventions for `name` attribute: `main_translation_update`, `updateMainTranslation`. Can you choose one convention and follow it?

Reply:
Another thing is, that for better code readability you could cache there result of `doc.querySelectorAll('input[name="updateMainTranslation"]')` in a variable like: `const inputs`. Then you won't have to move the `input.addEventListener...` part to the next line.